.eslintrc

```diff
@@ -12,6 +12,11 @@
   "globals": {
     "wp": "readonly"
   },
+  "settings": {
+    "jsdoc": {
+      "mode": "typescript"
```

Comment:
Allows doing `@property {import('./editor').` in JSDoc [like here](https://github.com/studiopress/genesis-custom-blocks/pull/51/files#diff-84d96f93e4494795fbab4f77ff96fe295b0e062e868b2f5b7498ac724b1af5aeR24)